internal/db/sqltest/tests/session/session_state_invalid_transition_test.sql

```diff
@@ -0,0 +1,16 @@
+begin;
+    select plan(4);
+
+    -- Ensure session state table is populated
+    select is(count(*), 1::bigint) from session_state where session_id = 's1_____clare' and state='pending';
+    select is(count(*), 1::bigint) from session_state where session_id = 's1_____cindy' and state='terminated';
+
+    -- Invalid duplicate state
+    select throws_ok($$ insert into session_state  ( session_id, state )
+            values  ('s1_____clare','pending')$$);
+
+    -- Invalid state transition from terminated to pending
+    select throws_ok($$ insert into session_state  ( session_id, state)
+    values  ('s1______cindy','pending')$$);
```

Comment:
now that this pattern is set, can you add additional tests for the other invalid scenarios, ie:

- terminated -> active
- terminated -> canceling 
- terminated -> terminated
- canceling -> active
- etc...